packages/remix-react/errorBoundaries.tsx

```diff
@@ -146,7 +147,8 @@ export function BoundaryShell({
       </head>
       <body>
         <main style={{ fontFamily: "system-ui, sans-serif", padding: "2rem" }}>
-          {contents}
+          {children}
+          {renderScripts ? <Scripts /> : null}
```

Comment:
We can't blindly render `<Scripts/>` in the default `HydrateFallback` since if the user has their own `Layout` we'll get two `<Scripts/>` instances.  So we only want to render it when they don't have a `Layout`.